java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -3670,7 +3670,7 @@ protected ArgsArray buildArgs(String... stringArgs) {
         ArgsArray.Builder commandArgs = ArgsArray.newBuilder();
 
         for (String string : stringArgs) {
-            commandArgs.addArgs(string);
+            commandArgs.addArgs(com.google.protobuf.ByteString.copyFromUtf8(string));
```

Comment:
You can add an import to shorten this
```suggestion
            commandArgs.addArgs(ByteString.copyFromUtf8(string));
```
or shorten more with static import
```suggestion
            commandArgs.addArgs(copyFromUtf8(string));
```

Reply:
Fixed